IndicoV2.Tests/Client/Submissions/SubmissionsClientTests.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using IndicoV2.Tests.TestUtils;
+
+namespace IndicoV2.Tests.Client.Submissions
+{
+    [TestClass]
+    public class SubmissionsClientTests
```

Comment:
Unit tests for new Adapter class. We're still thinking how to test that the best way.